include/swift/AST/Attr.def

```diff
@@ -272,6 +272,8 @@ DECL_ATTR(_implements, Implements,
           | NotSerialized | UserInaccessible,
           /* Not serialized */ 67)
 
+DECL_ATTR(NSKeyedArchiveLegacy, NSKeyedArchiveLegacy, OnClass, 68)
```

Comment:
This one should be `NotSerialized` too. (I realize that rules out adding the attribute on the class and the conformance in another module, but that's okay.)

Also `LongAttribute`, to improve printing.